### PR TITLE
If schema is empty, accept empty body (fixes mozilla-services/kinto#63)

### PR DESCRIPTION
**Pull request by leplatrem**
While I was writing this, I was strongly thinking in changing the default behaviour (schemaless by default).

Since this would be out of scope for this PR, I didn't change anything apart from fixing mozilla-services/kinto#63

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -11,6 +11,10 @@ This document describes changes between each past release.
 - Provide the userid when calling the hello page with an Authorization
   header. (#319)
 
+**Bug fixes**
+
+- ``data`` is not mandatory in request body if the resource does not define
+  any schema (fixes mozilla-services#63)
```
Comment (leplatrem) [inline CHANGELOG.rst]: typo: missing kinto

File: cliquet/schema.py [hunk 2]
```diff
@@ -61,6 +61,12 @@ def is_readonly(self, field):
         """
         return field in self.get_option("readonly_fields")
 
+    @classproperty
+    def missing(cls):
+        if cls != ResourceSchema:
```
Comment (Natim) [inline cliquet/schema.py]: Actually I would remove this sanity check.
If there is a schema, the missing properties in the schema will take care of raising an error (see /buckets/id/groups/id for instance) We cannot assume that only ResourceSchema accepts empty data.

Comment (Natim) [inline cliquet/schema.py]: I tried and it doesn't check for the missing properties. Is it possible to set {} as the default value for data but test that {} meets the requirements of the defined schema?

**Comment by leplatrem** [comment]
Updated :)

**Comment by Natim** [comment]
Thanks, works for me.

**Pull request merged**